client/src/app/components/BrowseOpportunities.js

```diff
@@ -116,6 +116,7 @@ export default function BrowseOpportunities(props) {
             <OpportunityCard
               key={`people-list-item-${index}`}
               data={opp}
+              isBrowsing
```

Comment:
is this declared somewhere in the file?

Reply:
isBrowsing is a prop for OpportunityCard. If this is included, then it reads as isBrowsing to be true, but if this is not included, then isBrowsing would be undefined or basically read as false. It is declared as a prop in OpportunityCard.js

Reply:
did you want to set a default value in browseOpportunities? what is the variable isBrowsing used for?

Reply:
Oh it's just to determine whether the card is for the browsing page or not. There's probably a better way to do it but this is what came to me and it was really easy to implement. Basically, if it has "isBrowsing," the invite will appear as an option and if it the variable isn't there, it won't appear. So like the cards in the browsing page have the invite button but the cards in the opportunity tab dont have them.

Reply:
I guess the default would be false since not including it would just read as undefined or false. If you want it to be more explicit, i can change it to isBrowsing={true} and when the card is not for the browsing page, you change it to isBrowsing={false}.

Reply:
I think a better solution would be to have a check in the OpportunityCard file.

Since you're working on the edit profile, I'm going to do the following:

- rename "Invite" to "Request to Join" on the OpportunityCard
- remove isBrowsing
- if I own the opportunity: don't show "Request to Join"
- if I don't own the opportunity: show "Request to Join"